README.md

```diff
@@ -32,7 +32,13 @@ $ mbedgt --version
 You can also install it manually via pip.
 
 ```
-pip install mbed-greentea
+sudo -H pip install mbed-greentea
```

Comment:
`sudo` is not available on every platform. Could you add instructions for platforms that don't have `sudo`? Could you explain why `-H` is used?